charts/sgx/values.yaml

```diff
@@ -0,0 +1,16 @@
+image:
+  hub: intel
+  tag: "devel"
+  pullPolicy: IfNotPresent
```

Comment:
are these used for anything?

Reply:
You are right, these are not needed in the SGX chart.